packages/common/src/types/savedCharts.ts

```diff
@@ -134,8 +134,10 @@ export const isPivotReferenceWithValues = (
 export type MarkLineData = {
     yAxis?: string;
     xAxis?: string;
-    name: string;
-    value: string;
+    name?: string;
+    value?: string;
+    type?: string;
+    uuid: string;
```

Comment:
iirc the reason for using `name / value` instead of uuid is because we had to rely on the properties of `markLine` to be able to use the chartconfig to store the values. This seem to work though, so I'm not sure if that is a requirement any more. 

https://echarts.apache.org/en/option.html#series-line.markLine

Reply:
Yea, I wondered about this, but we don't seem to be using their types, so I figured I'd just try it. I don't see any errors...